metrics/metrics.go

```diff
@@ -80,6 +83,10 @@ func NewRegistry(cfg config.Metrics) (r Registry, err error) {
 
 // parsePrefix parses the prefix metric template
 func parsePrefix(tmpl string) (string, error) {
+	// Backward compatibility condition for old metrics.prefix parameter 'default'
+	if tmpl == "default" {
```

Comment:
Would be nice to test this in `TestParsePrefix` to avoid regressions. Could you please add that and then I'll merge it.